benchmarks/bench_regex_fsm.py

```diff
@@ -0,0 +1,35 @@
+from transformers import AutoTokenizer
+
+from outlines.caching import cache_disabled
+from outlines.fsm.regex import reduced_vocabulary
+from outlines.models.transformers import TransformerTokenizer
+
+from .common import ensure_numba_compiled
+
+tokenizer_uris = {
```

Comment:
Can we create synthetic datasets instead of pulling external data?

Reply:
Done!